manifests/agent.pp

```diff
@@ -275,11 +278,21 @@
     }
   }
 
-  if $facts['kernel'] == 'windows' and $manage_choco {
-    package { $zabbix_package_agent:
-      ensure   => $zabbix_version,
-      provider => $zabbix_package_provider,
-      tag      => 'zabbix',
+  if $facts['kernel'] == 'windows' {
+    if $manage_choco {
+      package { $zabbix_package_agent:
+        ensure   => $zabbix_version,
+        provider => $zabbix_package_provider,
+        tag      => 'zabbix',
+      }
+    } else {
+      package { $zabbix_package_agent:
+        ensure          => $zabbix_package_state,
+        tag             => 'zabbix',
+        provider        => $zabbix_package_provider,
+        source          => $zabbix_package_source,
+        install_options => "SERVER=${server}",
```

Comment:
Is this needed, since we are managing the config file. I suppose that sets the Zabbix server and saves it to the configuration file?

Reply:
Maybe i'm wrong and we don't manage the configuration on windows :sweat_smile: 
If it's needed though you need to specify the `server` parameter, can you update the documentation?

Reply:
It is indeed needed, the MSI installer requires it when running non-interactive. We don't need to specify if though, the default parameter `$zabbix::params::agent_server` should be fine?

Reply:
I mean it should be fine as it point to localhost. Maybe we should document that providing a `server` parameter will automatically set the server IP.

Reply:
AFAIK this MSI parameter is only needed to complete the installation, its value is always be replaced in `zabbix_agent.conf` when converging; this commit doesn't really change how the `server` parameter is used from a user/documentation standpoint.